server.js

```diff
@@ -1,30 +1,90 @@
-import express from "express";
-import cors from "cors";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
-
-// Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
-
-// Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+import express from 'express'
+import cors from 'cors'
+import expressListEndpoints from 'express-list-endpoints'
+
+import topMusicData from './data/top-music.json'
+
+const port = process.env.PORT || 8080
+const app = express()
+
+//middleware
+app.use(cors())
+app.use(express.json())
+
+//routes
+
+//showing documentation of endpoints
+app.get('/', (req, res) => {
+	const endpoints = expressListEndpoints(app)
+	res.json(endpoints)
+})
+
+//route that shows all data from jsonfile
+app.get('/songs', (req, res) => {
+	res.json(topMusicData)
+})
+
+//route to find and show song by id
+app.get('/songs/:songId', (req, res) => {
```

Comment:
Nice to make the songId param connected to the songs endpoint. Makes perfect sense for me when thinking about how the user will probably use the frontend. 🥳